FIXED ||| ncurses fails

It installs after disabling lto in sys-config/...workarounds

`!!! Please attach the following file when seeking support:
!!! /var/tmp/portage/sys-libs/ncurses-6.0-r1/work/ncurses-6.0-abi_x86_32.x86/ncursestw/config.log
 *               ebuild.sh, line  124:  Called src_configure
 * ERROR: sys-libs/ncurses-6.0-r1::gentoo failed (configure phase):
configure: error: Cannot link test program for libdl
 * Call stack:
 *   econf failed
 *             environment, line 3670:  Called multilib-minimal_src_configure
configure: error: Cannot link test program for libdl

!!! Please attach the following file when seeking support:
!!! /var/tmp/portage/sys-libs/ncurses-6.0-r1/work/ncurses-6.0-abi_x86_64.amd64/ncursesw/config.log
 *             environment, line 3670:  Called multilib-minimal_src_configure
 *               ebuild.sh, line  124:  Called src_configure
 * ERROR: sys-libs/ncurses-6.0-r1::gentoo failed (configure phase):
`

Strange, we actually include a workaround for this in `ltoworkarounds.conf`:

~~~
sys-libs/ncurses LDFLAGS+=-ldl # checking whether able to link to dl*() functions... configure: error: Cannot link test program for libdl
~~~

I just tried re-emerging it myself and could not reproduce this bug (with the workaround applied).  Are you updated to the latest config?

--- a/sys-config/ltoize/files/package.cflags/ltoworkarounds.conf
+++ b/sys-config/ltoize/files/package.cflags/ltoworkarounds.conf
@@ -48,6 +48,7 @@ sys-libs/libcxxabi *FLAGS-=-flto* # /var/tmp/portage/sys-libs/libcxxabi-4.0.1/te
 games-emulation/parallel-n64-libretro *FLAGS-=-flto*
 games-emulation/mupen64plus-libretro *FLAGS-=-flto*
 net-p2p/cpuminer-opt *FLAGS-=-flto*
+x11-drivers/xf86-video-intel *FLAGS-=-flto*
 
 sys-process/criu *FLAGS-=-pipe* *FLAGS-=-flto* *FLAGS-="-fuse-linker-plugin" LDFLAGS-="-Wl,--hash-style=gnu" *FLAGS-="${GRAPHITE}" # Fewer settings may be possible. Needs testing. (Dependency of LXC.)
 

